src/dscanner/analysis/imports_sortedness.d

```diff
@@ -125,8 +135,8 @@ unittest
 		import d.a;
 		import d; // [warn]: %s
 	}c.format(
-		ImportSortednessCheck.MESSAGE,
-		ImportSortednessCheck.MESSAGE,
+		"The imports are not sorted in alphabetical order",
```

Comment:
Why not use "ImportSortednessCheck.Message" ? Is this change necessary?

Reply:
I would say yes, because now `ImportSortednessCheck` is not a class but a template. If we want to have the same style I could use something like `ImportSortednessCheck!ASTBase ...`, but I would say it's better as it is right now.

Reply:
Why not simply: `Message`? There's no need for the full name.

Reply:
Ah, sure I can do that, I just did not give it that much thought

Reply:
Is this good to go after applying this change @RazvanN7 ?

Reply:
Yes.

Reply:
I addressed that but with a slightly different approach that we also have for other tests as well, that I find more straightforward